src/index.js

```diff
@@ -341,11 +345,13 @@ function MarkdownEditor(props) {
     const { children, mark, attributes } = props;
 
     switch (mark.type) {
+      case 'html':
```

Comment:
why was html moved from matching the `code` case to matching the ### bold case?

Reply:
This is a very good question.

Reply:
I thought the answer was obvious, so that HTML would be bold?

Reply:
but doesn't it make more sense to match the `code` style if it's html? I could be wrong though!

Reply:
I reverted back to being with `code`